src/config/dynamic-theme-fixes.config

```diff
@@ -2255,6 +2258,15 @@ body,
 
 ================================
 
+bet.com
```

Comment:
This site seems to be a redirection.

Reply:
@Gusted, echoing what @q1800 said, this is the actual, non-redirected website (at least within the US).